lib/rust/ensogl/examples/vector-editor/src/lib.rs

```diff
@@ -0,0 +1,165 @@
+//! Example scene showing the usage of built-in high-level shapes.
```

Comment:
Copied docs. The file seems to be unrelated to PR name, nor it's mentioned in the description. And I guess if we want to merge this file to develop, we must mark it as "TODO still in progress fixme".